R/model.R

```diff
@@ -17,10 +18,12 @@ run_simulation <- function(pop, parameters) {
   human <- create_human(states,
                         variables,
                         events)
+  if (is.null(renderer)) processes <- list()
+  if (!is.null(renderer)) processes <- list(renderer)
```

Comment:
As these are exclusive
```suggestion
  if (is.null(renderer)) {
    processes <- list()
  } else {
    processes <- list(renderer)
  }
```